pytest/tests/mocknet/helpers/neard_runner.py

```diff
@@ -150,6 +174,46 @@ def __init__(self, args):
         # is no need to block reading that when inside the update_binaries rpc for example
         self.lock = threading.Lock()
 
+    def _configure_neard_logs(self):
+        try:
+            os.mkdir(self.neard_logs_dir)
+        except FileExistsError:
+            pass
+
+        variables = {
+            '__neard_logs_dir__': f'{self.neard_logs_dir}',
+            '__neard_logs_file_name__': f'{self.neard_logs_file_name}',
+            '__neard_logs_file_size__': '100M',
+            '__neard_logs_max_file_count__': '99',
+            '__neard_logs_max_size__': '100000000000',  # 100G
+        }
+        logrotate_config = LOGROTATE_TEMPLATE
+        # Replace variables in the template
+        for var, value in variables.items():
+            logrotate_config = re.sub(re.escape(var), value, logrotate_config)
+        self.logrotate_config_path = f'{self.neard_logs_dir}/.neard_logrotate_policy'
+        logging.info(
+            f'Setting log rotation policy in {self.logrotate_config_path}')
+        with open(self.logrotate_config_path, 'w') as config_file:
+            config_file.write(logrotate_config)
+        self.logrotate_binary_path = shutil.which('logrotate')
+        if self.logrotate_binary_path is None:
```

Comment:
can we check this at the beginning and do the rest only if the binary is there?

Reply:
I chatted with Razvan and logrotate can be installed on Mac for local testing, so resolving this.